sdk/file_sdk/Common/file_format_base.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FILE_FORMAT_BASE_H
+#define FILE_FORMAT_BASE_H

```

Comment:
this define should match filename

---
In reply to: [134247719](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134247719) [](ancestors = 134247719)